api/base/serializers.py

```diff
@@ -280,6 +299,54 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
```

Comment:
I see now there are new *views* tests, which will test this indirectly. Still I think there is enough added logic here to warrant adding isolated unit tests for our `JSONAPISerializer`.